wai-extra/wai-extra.cabal

```diff
@@ -81,6 +81,11 @@ extra-source-files:
   ChangeLog.md
   README.md
 
+flag build-app
+  description:        Build app executable.
+  manual:             True
+  default:            False
```

Comment:
How about enabling this flag in the `stack.yaml` so that CI will build the executable?